src/KDSoapClient/KDSoapClientInterface.h

```diff
@@ -346,9 +346,46 @@ class KDSOAP_EXPORT KDSoapClientInterface
       */
     void setTimeout(int msecs);
 
+    /**
+     * @brief setSendHTTPActionParameter
+     * @param sendHTTPAction
+     * This option can be used to enable/disable the sending of the SOAP action parameter
+     * in the Content-Type HTTP header (in the case of SOAP 1.2 requests).
+     * It might be necessary to disable it for certain  SOAP server implementations
+     * in the case if the action is passed in the SOAP envelope's header with WS-Addressing.
+     */
+    void setSendHTTPActionParameter(bool sendHTTPAction = true);
```

Comment:
I'd remove the default value, since the member defaults to true, so calling `setSendHttpActionParameter()` basically does nothing.

Reply:
Makes sense.